azure-communication-ui/azure-communication-ui-demo-app/src/androidTest/java/com/azure/android/communication/ui/ExampleInstrumentedTest.kt

```diff
@@ -1,25 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.android.communication.ui
```

Comment:
This test is just a placeholder for a real test, and could cause problems for other projects if run outside "com.azure.android.communication.ui" e.g. in an integrated app.